activestorage/db/update_migrate/20191206030411_create_active_storage_variant_records.rb

```diff
@@ -1,5 +1,7 @@
 class CreateActiveStorageVariantRecords < ActiveRecord::Migration[6.0]
-  def change
+  def up
+    return if table_exists?(:active_storage_variant_records)
+
     create_table :active_storage_variant_records do |t|
```

Comment:
Let's change to 
```suggestion
    create_table :active_storage_variant_records, if_not_exists: true do |t|
```

Reply:
Oh right I had forgotten about this 👍 
Also works with as a reversible migration directly.